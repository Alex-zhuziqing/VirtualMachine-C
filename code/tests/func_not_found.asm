FUNC LABEL 1
    PRINT VAL 1
    RET
FUNC LABEL 2
    PRINT VAL 2
    RET
