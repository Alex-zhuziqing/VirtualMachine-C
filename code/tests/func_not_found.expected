Function 0 not found
